docs/spfx/publish-to-marketplace.md

```diff
@@ -0,0 +1,82 @@
+
+## Use SharePoint Framework v1.11 or newer
+
+We recommend that you use SharePoint Framework v1.11 or newer to build solutions that you want to publish to the marketplace. Starting from version 1.11, SharePoint Framework solution manifest has additional information about your organization and solution.
+
+In the solution's manifest (managed in the `package-solution.json` file) there is a new section named `developer` with the following properties:
+
+Attribute|Description|Mandatory
+---|---|---
+`name`|Name of the organization that built the application|Yes
+`websiteUrl`|URL of a website with additional information about the application|Yes
+`mpnId`|Microsoft Partner Network ID (more details on [MS Partner Network](https://docs.microsoft.com/en-us/partner-center/mpn-overview))|Yes (for solutions published to the marketplace)
+`privacyUrl`|Privacy statement URL|Yes
+`termOfUseUrl`|Terms of use URL|Yes
+
+Including this additional information allows you to clearly communicate information about your application towards your users and in the future should allow you to access additional usage data.
+
+The developer information is displayed when installing web parts in Microsoft Teams. Information about the application and the developer that is displayed in the marketplace is managed separately in the marketplace.
+
+> [!TIP]
+> If your application is built using an older version of the SharePoint Framework, you can easily upgrade it to SharePoint Framework v1.11 or newer using Office 365 CLI. For more information, visit [https://pnp.github.io/office365-cli/cmd/spfx/project/project-upgrade/](https://pnp.github.io/office365-cli/cmd/spfx/project/project-upgrade/).
+
+## Install applications from the marketplace in the tenant app catalog
+
+SharePoint Online allows you to create a tenant app catalog where you centrally manage applications available in your organization. Additionally, you can create site collection app catalogs, where you can deploy applications to specific sites.
+
+Applications that you acquire from the marketplace, you can install only in tenant app catalogs. After installing the application, you can decide whether you want to deploy it globally and make it available to all sites in your tenant or if you want to deploy it only to specific sites.
+
+For more information about managing SharePoint Framework solution in the app catalog visit: [https://docs.microsoft.com/en-us/sharepoint/use-app-catalog](https://docs.microsoft.com/en-us/sharepoint/use-app-catalog) and [https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog](https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog).
```

Comment:
Need to remove locale from URLs... let MSDocs redirect as appropriate for user.

Reply:
I saw the notification, but wasn't aware you could just drop the locale. Thanks for the tip! I'll update it!